.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
labels instruction would be good also, if we have a documentation for it at least, like a excel file

Reply:
Let's add these later, because we don't have labels yet :p